### ISSUE TITLE
The function "commands-allow-op" in the Config does not work

### ISSUE THREAD

**Issue by TonyOneZero** [lucko/LuckPerms#1250]
### **What I have used:**
LuckPerms v4.3.7 on a Spigot-Server 1.8.8
Luckperms is connected to a MongoDB

**Plugins:**
```shell
/pl
[15:03:22 INFO]: Plugins (1): LuckPerms
```

**Config:**
```shell
# Defines if "opped" players should be able to use all LuckPerms commands by default.
#
# - Set to false to only allow users who have the permissions access to the commands
commands-allow-op: false
```

### **What I did:**
I set up a new server and connected LuckPerms to an empty Mongo database. Then I set the setting "commands-allow-op" in the config to false. 

Then I tried to access the LuckPerms Commands without Op. Which of course didn't work.

```shell
/lp creategroup test
[LP] Running LuckPerms v4.3.7.
```
Then I tried to access the LuckPerms Commands with Op. What shouldn't actually work.

```shell
/lp creategroup test
[LP] test was successfully created.
```

### **The problem:**
I may misunderstand your function in the config. But if I understand it right it shouldn't work with Op.

**Comment by Andre601** [comment]
Did you actually restart the server, after you changed the settings?

**Comment by TonyOneZero** [comment]
> Did you actually restart the server, after you changed the settings?

Yes of course.

**Comment by lucko** [comment]
I can't reproduce using a totally new server.

![](https://i.imgur.com/zjrkZoJ.png)

Perhaps you assigned some permissions to yourself by mistake?

**Comment by TonyOneZero** [comment]
@lucko 

I tried again. Please change the following and try again.

```shell
# If the plugin should apply Bukkit default permissions.
#
# - Plugin authors can define permissions which should be given to all users by default, or setup
#   permissions which should/shouldn't be given to opped players.
# - If this option is set to false, LuckPerms will ignore these defaults.
apply-bukkit-default-permissions: false
```

**Comment by TonyOneZero** [comment]
It is not possible to leave LuckPerms out of this setting, because I want to use this setting.

**Comment by TonyOneZero** [comment]
@lucko
It is not possible to leave LuckPerms out of this setting, because I want to use this setting.

**Comment by lucko** [comment]
No, LP uses the "bukkit default permissions" system to implement the `commands-allow-op` setting.
